crates/rome_formatter/src/lib.rs

```diff
@@ -565,6 +568,13 @@ pub enum FormatError {
 
     /// In case printing the document failed because it has an invalid structure.
     InvalidDocument(InvalidDocumentError),
+
+    /// Formatting failed because formatting some content encountered a situation where a layout
+    /// choice by an enclosing object resulted in a poor layout for the child object.
```

Comment:
```suggestion
    /// chosen by an enclosing node, resulted in a poor layout for the child node.
```
When I read "object" I think the actual objects in JS... maybe node is wider term. I thought about "element" but in can be mistaken for HTML elements.

Reply:
I prefer not to use `element` to avoid confusing them with `FormatElement`. I'll probably use `type` to be consistent with `Format` and add a link to `Format`.